feat(about) Check for latest release if user is admin

steffen-weber opened the pull request:
fix #236

maximilianhuber commented on src/www/ui/page/AboutPage.php:
```diff
@@ -50,15 +50,49 @@ function __construct()
+      $latestRelease = $this->curlGit('releases/latest');
+      $since = '?since=' . date('Y-m-dH:i:sZ', time() - 3600 * 24 * 30);
+      $commits = $this->curlGit('commits' . $since);
+      $commit = substr($commits[0]['sha'],0,6);
```
When visiting `http://localhost:8081/repo/?mod=about` with enabled debug information I get the warning
```
Notice: Undefined offset: 0 in /usr/local/share/fossology/www/ui/page/AboutPage.php on line 64
```

--

![tmp3](https://cloud.githubusercontent.com/assets/1187050/14456931/c80bbc62-00a7-11e6-909a-0a751dfb56d3.png)

maximilianhuber commented:
You could also add links to https://github.com/fossology/fossology/ and https://github.com/fossology/fossology/releases/latest (behind the text The latest version 3.0.0)

maximilianhuber commented:
@steffen-weber: the merged PRs from **shaheemazmalmmd** have introduced merge-conflicts to your PR. Sorry.

steffen-weber commented:
Got same error when connectivity was bad. So existence of both infos (release/commit) are checked now separately.
-> restructured the feature, added tests, rebased onto master
